include/teakra/teakra.h

```diff
@@ -22,14 +22,13 @@ class Processor;
 
 class Teakra {
 public:
-    Teakra();
+    Teakra(bool use_jit);
```

Comment:
Likely preferrable to do `bool use_jit = false` to not break the build of any unsuspecting folk once this is merged upstream. It's likely some people we're not aware of also use Teakra.